BraintreeDropIn/BTDropInOverrides.h

```diff
@@ -0,0 +1,8 @@
+#import <Foundation/Foundation.h>
+
+@interface BTDropInOverrides : NSObject
+
+/// Used for testing. Show Venmo option when configured correctly but Venmo is not installed.
+@property (class, nonatomic) bool displayVenmoOption;
```

Comment:
Nitpick but should be `BOOL` instead of `bool` I think as well as in `BTDropInOverrides.m`.